<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="fme_card_content_0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QFrame {
	border : 1px solid;
	border-color: #CED7E6;
    border-top-right-radius: 7px;
	border-top-left-radius: 7px;
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="fme_image_cap_2">
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color : #CED7E6;
	border : 1px solid;
	border-color: #CED7E6;
    border-top-right-radius: 7px;
	border-top-left-radius: 7px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_19" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="fme_image">
           <property name="styleSheet">
            <string notr="true">QFrame {
    background-image: url(:/images/assets/img/image-polaroid.png);
    background-repeat : none;
	background-position: center center;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QFrame" name="frame">
              <property name="styleSheet">
               <string notr="true">QFrame {
	background-color : transparent;
	background-image : url();
border : none;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btn_remove">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : red;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : red;
	border: 1px solid red;
	border-radius: 5px;
}

</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="assets.qrc">
                   <normaloff>:/icon/assets/icon/square-xmark1.png</normaloff>:/icon/assets/icon/square-xmark1.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>35</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="fme_card_content_2">
        <property name="styleSheet">
         <string notr="true">QFrame {
	border : 1px solid;
	border-color: #CED7E6;
    border-top-right-radius: 0px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 7px;
	border-bottom-left-radius: 7px;
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
	border : none;
	border-color: transparent;
    border-top-right-radius: 0px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_21" stretch="0">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lbl_card_title">
              <property name="styleSheet">
               <string notr="true">QLabel {
	margin-top : 12px;
	font : 18px &quot;Arial&quot;;
	color: #34373B;
}</string>
              </property>
              <property name="text">
               <string>Product Name</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
	border : none;
	border-color: transparent;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lbl_total">
              <property name="styleSheet">
               <string notr="true">QLabel {
	font : 24px &quot;Arial&quot;;
	color: green;
}</string>
              </property>
              <property name="text">
               <string>Total: $555,55</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_11">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <property name="spacing">
                <number>8</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QSpinBox" name="sbx_amount">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit, QTextEdit, QComboBox, QSpinBox, QDoubleSpinBox {
	background-color : transparent;
	border : 1px solid #DADDE1;
	border-radius : 5px;
	color : #616161;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_action">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : orange;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : orange;
	border: 1px solid orange;
	border-radius: 5px;
}

</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="assets.qrc">
                   <normaloff>:/icon/assets/icon/cart-shopping3.png</normaloff>:/icon/assets/icon/cart-shopping3.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_10">
              <property name="styleSheet">
               <string notr="true">QFrame {
	background-color : transparent;
	border-image : none;
	border : none;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btn_save">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="toolTip">
                  <string>Mostrar productos</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : #1877F2;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : #1877F2;
	border: 1px solid #1877F2;
	border-radius: 5px;
}
</string>
                 </property>
                 <property name="text">
                  <string>Guardar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
